### PR TITLE
Update grammar

### PR DESCRIPTION
**Pull request by ice1000**
bors r+

### CODE REVIEW

File: src/main/java/org/aya/intellij/parser/AyaParserDefinition.java [hunk 1]
```diff
@@ -42,26 +32,6 @@ public class AyaParserDefinition extends SyntaxHighlighterFactory implements Par
...
-   * it back via: {@link PsiFile#getNode()}
-   */
-  @Override public @NotNull PsiFile createFile(@NotNull FileViewProvider viewProvider) {
-    return new AyaPsiFile(viewProvider);
-  }
-
-  @Override public @NotNull PsiElement createElement(@NotNull ASTNode node) {
-    return AyaPsiElementTypes.Factory.createElement(node);
-  }
-
```
Comment (imkiva) [inline src/main/java/org/aya/intellij/parser/AyaParserDefinition.java:64]: Why delete these? They are required by IJ

File: src/main/java/org/aya/intellij/parser/AyaParserDefinition.java [hunk 2]
```diff
@@ -5,35 +5,25 @@
...
 
   public static @NotNull Lexer createLexer() {
     return new FlexAdapter(new _AyaPsiLexer());
   }
 
-  @Override
-  public @NotNull SyntaxHighlighter getSyntaxHighlighter(@Nullable Project project, @Nullable VirtualFile virtualFile) {
-    return new SyntaxHighlight();
-  }
-
```
Comment (imkiva) [inline src/main/java/org/aya/intellij/parser/AyaParserDefinition.java:36]: Then there should be a new `SyntaxHighlighter` in plugin.xml

**Review by imkiva — COMMENTED** [review]
This would break plugin features.

File: src/main/java/org/aya/intellij/parser/AyaParserDefinition.java (hunk 1 above)
Comment (ice1000) [reply src/main/java/org/aya/intellij/parser/AyaParserDefinition.java:64]: Oh my god I didn't mean to I used copy and paste w/o realizing the differences

File: src/main/java/org/aya/intellij/parser/AyaParserDefinition.java (hunk 2 above)
Comment (ice1000) [reply src/main/java/org/aya/intellij/parser/AyaParserDefinition.java]: :sob:

**Comment by ice1000** [comment]
QAQ

**Comment by ice1000** [comment]
bors r_

**Comment by ice1000** [comment]
bros r+

**Comment by ice1000** [comment]
不玩了

File: src/main/java/org/aya/intellij/parser/AyaParserDefinition.java (hunk 1 above)
Comment (imkiva) [reply src/main/java/org/aya/intellij/parser/AyaParserDefinition.java]: LOL

Comment (imkiva) [reply src/main/java/org/aya/intellij/parser/AyaParserDefinition.java]: We should decouple ParserDefinition and use the minimal one in upstream and the extended one in IJ plugin.

**Comment by imkiva** [comment]
image

I kinda think parsing test is redundant in this repo (given that we will move to GK parser).

**Pull request merged**